src/components/gamma/SplitViewScreen.tsx

```diff
@@ -2,11 +2,9 @@ import React from 'react';
 import { StyleSheet } from 'react-native';
 import type { ViewProps } from 'react-native';
 import SplitViewScreenNativeComponent from '../../fabric/gamma/SplitViewScreenNativeComponent';
-import type { NativeProps } from '../../fabric/gamma/SplitViewScreenNativeComponent';
 
-export type SplitViewScreenNativeProps = NativeProps & {
-  // Overrides
-};
+// eslint-disable-next-line @typescript-eslint/ban-types
+export type SplitViewScreenNativeProps = {};
```

Comment:
`{}` does not denote empty object in typescript. I think we can remove this type entirely for now, if it is unused (it should not be used outside of this source file anyway).